docs/upgrade.html

```diff
@@ -75,7 +75,7 @@ <h5><a id="upgrade_320_notable" href="#upgrade_320_notable">Notable changes in 3
         <li>Kafka has replaced log4j with reload4j due to security concerns.
             This only affects modules that specify a logging backend (<code>connect-runtime</code> and <code>kafka-tools</code> are two such examples).
             A number of modules, including <code>kafka-clients</code>, leave it to the application to specify the logging backend.
-            More information can be found at <a href"https://reload4j.qos.ch">reload4j</a>.
+            More information can be found at <a href="https://reload4j.qos.ch">reload4j></a>.
```

Comment:
nit: The `>` after `reload4j` does not seem right.

Reply:
Ah, HTML is hard! Fixed, thanks